### DISCUSSION TITLE
Star Rating plugin for Tailwind (stars input)

### DISCUSSION THREAD

**Discussion by cyggarro** [mdbootstrap/TW-Elements#1528] [Share ideas & request features]
This component should include the ability to customize the number of stars displayed, the size of the stars, the color of the stars (both for filled and empty stars), and the ability to set the default rating value.

**Comment by filipkappa** [comment]
Thanks for submitting this idea!

It gathered so much traction that we have added a task for this feature: https://github.com/mdbootstrap/Tailwind-Elements/issues/1560

You can track our progress live here:
https://tailwind-elements.com/docs/standard/getting-started/changelog/

**Comment by filipkappa** [comment]
We added this to Tailwind Elements! You can now use the feature:
https://tailwind-elements.com/docs/standard/components/rating/
